proto/penumbra/penumbra/view/v1alpha1/view.proto

```diff
@@ -115,8 +134,20 @@ message AuthorizeAndBuildRequest {
 }
 
 message AuthorizeAndBuildResponse {
-  // The transaction that was built.
-  core.transaction.v1alpha1.Transaction transaction = 1;
+  // Signals that building is in progress.
+  message BuildProgress {
+    // An approximate progress of the build, from 0 to 1.
+    float progress = 1;
```

Comment:
Does this correlate to the % of actions that have been built thus far?

Reply:
It's up to the implementation, I guess, in principle an implementation that had insight into how much proving was done as it was happening could give better updates. I believe there's a way to get Rayon to do this on the Rust side. I think at the proto level we should be as agnostic as possible, and initially, we should just try to expose info we have rather than adding progress plumbing into the proving internals.